app/src/main/java/com/iic/shopingo/dal/models/BaseRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.iic.shopingo.dal.models;
+
+/**
+ * Created by assafgelber on 3/8/15.
+ */
+public class BaseRequest {
+  public enum RequestStatus {
+    PENDING,
+    ACCEPTED,
+    DECLINED,
+    CANCELED
+  }
+
+  private Contact contact;
+  private ShoppingList shoppingList;
+  private RequestStatus status;
+
+  public BaseRequest(Contact contact, ShoppingList shoppingList, RequestStatus status) {
+    this.contact = contact;
```

Comment:
I think contact shouldn't be in base.
You should have `shopper` in `OutgoingRequest` and `requester` in `IncomingRequest`.